src/types/gecko-profile.js

```diff
@@ -267,17 +267,14 @@ export type GeckoCounter = {|
   name: string,
   category: string,
   description: string,
-  sample_groups: $ReadOnlyArray<{|
-    id: number,
-    samples: {|
-      schema: {|
-        time: 0,
-        number: 1,
-        count: 2,
-      |},
-      data: $ReadOnlyArray<[number, number, number]>,
+  samples: {|
+    schema: {|
+      time: 0,
+      number: 1,
+      count: 2,
```

Comment:
I wonder if now might be a good time to swap `number` and `count` to have `number` as the last value so the gecko profiler could skip emitting it when the counter only produces counts (that's the case of all counters but the memory ones).

Reply:
Actually, this was already done on the gecko side: https://bugzilla.mozilla.org/show_bug.cgi?id=1835649 I will just update the type.